src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/DiagnosticSourceEventSource.cs

```diff
@@ -96,7 +96,7 @@ namespace System.Diagnostics
     /// 
     /// * How data is logged in the EventSource 
     /// 
-    /// By default all data from Diagnostic sources is logged to the the DiagnosticEventSouce event called 'Event' 
+    /// By default all data from Diagnostic sources is logged to the DiagnosticEventSouce event called 'Event' 
```

Comment:
The latter, with the type name being used, seems most reasonable and in keeping with the rest.